emulator/src/extensions/fuzzer/ext_fuzzer.c

```diff
@@ -60,6 +62,8 @@ static bool fuzzerInitCb(emuArgs_t* emuArgs)
     fuzzer.motion  = emuArgs->fuzzMotion;
     fuzzer.time    = emuArgs->fuzzTime;
 
+    fuzzer.buttonMask = (PB_A | PB_B | PB_UP | PB_DOWN | PB_LEFT | PB_RIGHT | PB_START);
```

Comment:
I think in the past we used `--lock` when fuzzing to avoid the `PB_SELECT` problem. This works too, and opens the opportunity to make it configurable maybe one day.

Reply:
Yeah, the feedback I got from Thaeli that prompted this was that these days `--lock` doesn't really help fuzz a specific mode much, the fuzzer ends up spending most of its time opening and closing quick settings and changing the brightness instead.